examples/great_expectations_data_validation/run.py

```diff
@@ -12,8 +12,15 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
-
 from pipelines.validation import validation_pipeline
 
+from zenml.client import Client
+
 if __name__ == "__main__":
     validation_pipeline()
+
+    c = Client()
+    orchestrator = c.active_stack.orchestrator
+    orchestrator_config = orchestrator.config
+    if orchestrator_config.get("kubeflow_hostname"):
+        print("Kubeflow URL: `{orchestrator_config.kubeflow_hostname}`")
```

Comment:
![image](https://github.com/zenml-io/zenml/assets/4792082/a87d6a7f-b174-44b7-90f2-56c8c21b23b2)
via https://docs.zenml.io/user-guide/starter-guide/fetch-runs-after-execution

I think we can use the post execution for this